KsApi/models/Reward.swift

```diff
@@ -5,6 +5,7 @@ import Runes
 
 public struct Reward {
   public let backersCount: Int?
+  public let convertedMinimum: Double
```

Comment:
Is `convertedMinimum` guaranteed to always be returned? If no conversion is required (ie. project's currency is the same as the user's currency) is the `convertedMinimum` just the same as the `minimum`?

Reply:
Exactly